specification/datadog/resource-manager/Microsoft.Datadog/stable/2022-08-01/datadog.json

```diff
@@ -0,0 +1,2013 @@
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The ARM id of the resource.",
+          "type": "string"
+        },
+        "sendingMetrics": {
+          "description": "Flag indicating if resource is sending metrics to Datadog.",
+          "type": "boolean"
+        },
+        "reasonForMetricsStatus": {
+          "description": "Reason for why the resource is sending metrics (or why it is not sending).",
+          "type": "string"
+        },
+        "sendingLogs": {
+          "description": "Flag indicating if resource is sending logs to Datadog.",
+          "type": "boolean"
+        },
+        "reasonForLogsStatus": {
+          "description": "Reason for why the resource is sending logs (or why it is not sending).",
+          "type": "string"
+        }
+      }
+    },
+    "MonitoredResourceListResponse": {
+      "description": "Response of a list operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Results of a list operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MonitoredResource"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "The object that represents the operation.",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Service provider, i.e., Microsoft.Datadog.",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Type on which the operation is performed, e.g., 'monitors'.",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Operation type, e.g., read, write, delete, etc.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of the operation, e.g., 'Write monitors'.",
+          "type": "string"
+        }
+      }
+    },
+    "OperationResult": {
+      "description": "A Microsoft.Datadog REST API operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name, i.e., {provider}/{resource}/{operation}.",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "type": "boolean"
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of GET request to list the Microsoft.Datadog operations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of operations supported by the Microsoft.Datadog provider.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationResult"
```

Comment:
Consider to add x-ms-identifiers extension for array properties. [http://azure.github.io/autorest/extensions/#x-ms-identifiers](http://azure.github.io/autorest/extensions/#x-ms-identifiers)

Reply:
Point taken. Since this will be a breaking change, we'll raise a separate PR just to accommodate all the changes mentioned by you. Currently, the purpose of the PR is to accommodate multi-subscription changes.